docs/config.adoc

```diff
@@ -1204,10 +1204,10 @@ activate.
 
 There are two additional variants of `+tap-hold+`:
 
-* `+tap-hold-press+`
+* `+tap-hold-press+` or `+tap🠿+`
```

Comment:
For me at least, the 🠿 and 🠽 unicode characters are difficult for me to distinguish. Some random alternative thrown out there:

```
tap⮱↑
tap⮱↓
tap⮱↑timeout
tap⮱↓timeout
tap⮱🗙
```

Reply:
could also add the heavier variants
🡃 🡁 vs 
🠿🠽 or
🠻 🠹

btw, is there a trick to match by elements in an array in Rust so you could easier add a variant by adding it to an array `["down","🠿"]` rather than having the whole new variable?

Reply:
> btw, is there a trick to match ...

I'm not aware of any trick to make this trivially easier

Reply:
>could also add the heavier variants
🡃 🡁 vs
🠿🠽 or
🠻 🠹

For me these are all much more difficult to distinguish than the thin arrows.

Reply:
Yeah, they are, but the thin arrows risk being confused with regular taps

A few other ideas otherwise will add those ⮱ as an extra option

↓↑ regular for comparison
⯯⯭
↡↟
☟☝
⭳⭱ think only these can be mapped to "being held"
↓↑ regular for comparison

Reply:
The thin arrows are being used for press/release in, so it seems appropriate to use them for consistency in tap-hold as well:

```
  os2 (one-shot↓ 2000 lsft)
  os3 (one-shot↑ 2000 lctl)
```

Perhaps the `hold` is not worthwhile omitting at all, as with `shot` in `one-shot`.

I would go with either

```
tap-hold↑
tap-hold↓
tap-hold↑timeout
tap-hold↓timeout
tap-hold🗙
```

or

```
tap⮱↑
tap⮱↓
tap⮱↑timeout
tap⮱↓timeout
tap⮱🗙
```

Reply:
Re. `one-shot` I'm just not aware of any good symbol, otherwise also would've added it :)

and for hold don't remeber seeing a good "pressed key" non-arrow unicode symbol, otherwise could use that plus ↓↑, that would indeed be better

Reply:
This would look like a held button (if it were located lower), but unfortunately can't find a great shorter symbol for that, despite the variety, unicode doesn't have good computer-related symbolic support

tap▬▭▬↓
tap▬▭▬↑

only substitute, but that could be confused with ␠␣ 
tap⏡↓
tap⏡↑

or with some imagination you could say the circle is your finger on top of a key
tap߽↓
tap߽↑

Reply:
Or actually since we don't need to differentiate anymore if we add explicit ↓↑, we can just use the same heavy arrow for a "hold" 

tap🠿↑
tap🠿↓
tap🠿↑timeout
tap🠿↓timeout
tap🠿🗙